### ISSUE TITLE
render comment as double line inline text

### ISSUE THREAD

**Issue by statementreply** [LingDong-/wenyan-lang#148]
It seems to be common to render author's notes as double line inline text with smaller font size in classical Chinese books. The space on the top of pages was more commonly used for reader's notes.

Example for comment rendering style:
![example comment rendering style](https://i.imgur.com/DloUpAN.png)

**Comment by LingDong-** [comment]
Yes! this feature is planned. Please see also my comment on https://github.com/LingDong-/wenyan-lang/pull/35. Thanks for the suggestion!

**Comment by heangfat** [comment]
我正欲言此事。
此廼雙行夾注。其例可參見[哂蟹齋](https://vert.neocities.org/)之[楚辭](https://vert.neocities.org/cb/criosi/)。此版式固應由 CSS 爲之，然 W3C 遟遟不行，故我以 Java 腳本自作之矣。
排版要點如下：爲求齊行，必用行間標點(即不佔位標點)，括號亦不例外。夾注字與正文字等長，而筆畫細于正文字。一樓所示例，注文筆畫粗埒正文，一朢正裨難辨，是爲一忌。
另，魔法注釋可用單行夾注或眉批爲之。

**Comment by heangfat** [comment]
注：[哂蟹齋](https://vert.neocities.org/)之雙行夾注用火狐廼得見。
